### PR TITLE
implemented button_map for render_form_row

### PR DESCRIPTION
**Pull request by PanderMusubi**

### CODE REVIEW

**Comment by PanderMusubi** [comment]
If needed, I can also extend the readthedocs documentation and/or the example application.

**Comment by greyli** [comment]
Sorry for the late response, please update the docs and the example application.

File: flask_bootstrap/templates/bootstrap/form.html [hunk 1]
```diff
@@ -246,13 +246,13 @@
...
 
-{% macro render_form_row(fields, row_class='form-row', col_class_default='col', col_map={}) %}
+{% macro render_form_row(fields, row_class='form-row', col_class_default='col', col_map={}, button_map={}) %}
 {#
 fields: an iterable of fields to render in a row.
 row_class: Class to apply to the div intended to represent the row, like
   'form-row' or 'row'.
 col_class_default: Default col class to apply if nothing more specific is said about a field.
-col_class_dict: A dictionary which keys are fields.name and which values are the col class to apply to the field.
+col_map: A dictionary which keys are fields.name and which values are the col class to apply to the field.
```
Comment (greyli) [inline flask_bootstrap/templates/bootstrap/form.html]: a line to describe "button_map" should be added to here.

Comment (PanderMusubi) [reply flask_bootstrap/templates/bootstrap/form.html]: See https://github.com/z25/bootstrap-flask/commit/a1799e071a41eff0ff3ef7a6e5b7bea2cbba67d5

**Comment by PanderMusubi** [comment]
If all is fine now, could you merge this one please? Thanks

**Comment by greyli** [comment]
Merged, thanks!

**Pull request merged**